edits to PGD CLI content (#4688)
EnterpriseDB/docs: product_docs/docs/pgd/5/cli/index.mdx

  description: "The PGD Command Line Interface (CLI) is a tool to manage your EDB Postgres Distributed cluster"
---

> The EDB Postgres Distributed Command Line Interface (PGD CLI) is a tool for managing your EDB Postgres Distributed cluster. It allows you to run commands against EDB Postgres Distributed clusters. It can be installed automatically on systems in a TPA-deployed PGD cluster. Or it can be installed manually on systems that can connect to any PGD cluster, including BigAnimal distributed high-availability PGD clusters or PGD clusters deployed using the EDB PGD for Kubernetes operator.

Reviewer: It is installed automatically on each node in a PGD cluster deployed using TPA.  You may want to install it manually to connect to a TPA-deployed cluster remotely.  

The PGD CLI needs to be installed manually for use with PGD clusters deployed to EDB BigAnimal or with the EDB Postgres Distributed for Kubernetes operator.

Reviewer: we probably need to agree to how we want to handle the BA references but there's a Cluster Type in the UI and the option is named Distributed High Availability so I don't think chaning them all to "distributed high-avialability" makes sense

> The EDB Postgres Distributed Command Line Interface (PGD CLI) is a tool for managing your EDB Postgres Distributed cluster. It allows you to run commands against EDB Postgres Distributed clusters. It can be installed automatically on systems in a TPA-deployed PGD cluster. Or it can be installed manually on systems that can connect to any PGD cluster, including BigAnimal distributed high-availability PGD clusters or PGD clusters deployed using the EDB PGD for Kubernetes operator.

Reviewer 2: ```suggestion
The EDB Postgres Distributed Command Line Interface (PGD CLI) is a tool for managing your EDB Postgres Distributed cluster. It allows you to run commands against EDB Postgres Distributed clusters. It is installed automatically on systems in a TPA-deployed PGD cluster. Or it can be installed manually on systems that can connect to any PGD cluster, such as EDB BigAnimal Distributed High Availability clusters or PGD clusters deployed using the EDB PGD for Kubernetes operator.
```

> See [Installing PGD CLI](installing_cli) for information about how to install PGD CLI automatically with Trusted Postgres Architect and manually.

Reviewer: I don't think it tells them how to install with TPA (just says again that it will be done) = the installing chapter is really about manually installing depending on how the PGD cluster was deployed

> See [Installing PGD CLI](installing_cli) for information about how to install PGD CLI automatically with Trusted Postgres Architect and manually.

Reviewer 2: ```suggestion
See [Installing PGD CLI](installing_cli) for information about how to manually install PGD CLI on systems.
```
